tests/test_neurips_conference.py

```diff
@@ -1020,6 +1024,74 @@ def test_setup_matching(self, conference, client, helpers):
         # print(url)
         # assert False
 
+    def test_ac_reassignment(self, conference, helpers, client):
+
+        pc_client=openreview.Client(username='pc@neurips.cc', password='1234')
+        submissions=conference.get_submissions()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 3
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 5
+
+        ac_assignment = pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment', head=submissions[0].id)[0]
+
+        ## Remove AC assignment
+        ac_assignment.ddate = openreview.tools.datetime_millis(datetime.datetime.utcnow())
+        pc_client.post_edge(ac_assignment)
+
+        helpers.await_queue()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 3
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 4
+
+        assert [] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Area_Chairs').members
+        assert [] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Senior_Area_Chairs').members
+
+        ## Add AC assignment
+        ac_assignment.ddate = None
+        pc_client.post_edge(ac_assignment)
```

Comment:
So if an AC is added/removed, its corresponding SAC is also added/removed?

Reply:
yes, they need to be synced. Otherwise who is the SAC for that submission?

Reply:
Oh yes, I still had the idea that SAC were directly linked to the submission